plexapi/base.py

```diff
@@ -89,6 +89,22 @@ def _buildItemOrNone(self, elem, cls=None, initpath=None):
         except UnknownType:
             return None
 
+    def _buildDetailsKey(self, **kwargs):
+        """ Builds the details key with the XML include parameters.
+            All parameters are included by default with the option to override each parameter
+            or disable each parameter individually by setting it to False or 0.
+        """
+        details_key = self.key
+        if hasattr(self, '_includes'):
+            includes = {}
+            for k, v in self._includes.items():
```

Comment:
This should probably be sorted, tests have broken because of the order before.

Reply:
I think you want `urlencode(sorted(includes.items()), doseq=True)` below instead. It doesn't matter if the dict is unsorted, but you want URL parameters in the `details_key` to be in the same order.